core/DefaultFilters.lua

```diff
@@ -372,12 +366,14 @@ function addon:SetupDefaultFilters()
 				class, subclass = TRADE_GOODS, class
 			end
 
+			--TODO: Implement an option to override `subclass` with professions from our `addon.TRADESKILL_MAP`?
 			local reagentData = addon.ItemDatabase:ReagentData(slotData)
-
 			if self.db.profile.splitBySubclass[class] then
-				return (self.db.profile.splitExpansion and reagentData and reagentData.expansionName .. " - " ..subclass) or (reagentData and reagentData.profession) or subclass, class
+				return (self.db.profile.splitExpansion and reagentData and subclass.." ("..reagentData.expacName..")")
+				or subclass, class
 			else
-				return (self.db.profile.splitExpansion and reagentData and reagentData.expansionName .. " - " ..class) or class
+				return (self.db.profile.splitExpansion and reagentData and class.." ("..reagentData.expacName..")")
```

Comment:
nit: I don't like using `(text)` in the English language, as it subordinates one clause to another. I think either the previous `-` is more fitting, or perhaps `class: expansion`.

Reply:
Yes, either is fine by me. My preference would be for `class: expansion` or even `class, expansion`.